collections/sentinel-1-grd.yaml

```diff
@@ -221,6 +239,12 @@ Summaries:
         - - 0.00037037037
           - 0.00037037037
         unit: °
+      data_type: float32
+      statistics:
+        minimum:
+          - 0
+        maximum:
+          - 0.5
```

Comment:
Can you merge latest master to this branch? The `dataMask` band we added is missing here (when doing so please also check, I think we forgot to add the band at one spot to the list of available bands.

Reply:
Merged latest master and added `dataMask` to the list of bands.